package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "babel-node": "babel-node --presets=/*a*/",
-    "task1": "nodemon ./task1.js localhost 8080",
-    "task2": "nodemon ./task2.js localhost 8080",
-    "task3": "nodemon --exec babel-node -- ./task3.js"
+    "task1": "nodemon ./src/task1.js localhost 8080",
+    "task2": "nodemon ./src/task2.js localhost 8080",
+    "task3": "nodemon --exec babel-node -- ./src/task3.js",
+    "task4": "tsc && nodemon --exec babel-node -- ./dist/task4/task4.js"
```

Comment:
why you're running file from dist folder?

Reply:
Because my typescript files are compiled from src folder to disc folder